docs-src/community.md

```diff
@@ -24,7 +24,7 @@ questions, discussions.
 
 Meetings discussing the evolution of the service APIs will alternate times to
```

Comment:
grep caught this left-over reference to Service APIs.

Reply:
Updated, thanks!